classes/class.ilTestCronConfigGUI.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+require_once __DIR__ . "/../vendor/autoload.php";
+
+use iLUB\Plugins\TestCron\Helper\DIC;
+
+/**
+ * Class ilTestCronConfigGUI
+ *
+ *
+ * @package
+ */
+class ilTestCronConfigGUI extends ilPluginConfigGUI {
+
+	use DIC;
+
+    /**
+     * @inheritdoc
+     * @throws ilCtrlException
+     */
+	public function executeCommand() {
+
+		parent::executeCommand();
+		switch ($this->ctrl()->getNextClass()) {
```

Comment:
```suggestion
		if ($this->ctrl()->getNextClass()==strtolower(testCronMainGUI::class) {
		                 $h = new testCronMainGUI();
				$this->ctrl()->forwardCommand($h);
				return;
```

Reply:
Könnte man in diesem Fall so machen aber ilias-pattern ist ein Switch statement zu benutzten besonders weil später doch noch etwas dazukommen könnte/sollte.